test/lightning/attempts_test.exs

```diff
@@ -656,4 +656,30 @@ defmodule Lightning.AttemptsTest do
       assert reloaded_unfinished_step.finished_at != nil
     end
   end
+
+  describe "wipe_dataclip_body/1" do
+    test "clears the dataclip body and request fields" do
+      project = insert(:project)
+      dataclip = insert(:http_request_dataclip, project: project)
+
+      %{triggers: [trigger]} =
+        workflow = insert(:simple_workflow, project: project)
+
+      %{attempts: [attempt]} =
+        work_order_for(trigger, workflow: workflow, dataclip: dataclip)
+        |> insert()
+
+      assert dataclip.body
+      assert dataclip.request
+      refute dataclip.wiped_at
+
+      :ok = Attempts.wipe_dataclip_body(attempt)
```

Comment:
Please extend the test to cover the effects of `update_all` that are used in the implementation.

Reply:
@stuartc I've renamed the function to `wipe_dataclips`, but at the moment it only clears the given attempt's `dataclip`. Should I update the logic to update the associated `steps` dataclips' as well?